splice_machine/src/test/java/com/splicemachine/derby/test/framework/SpliceWatcher.java

```diff
@@ -377,4 +378,13 @@ public long getConglomId(String tableName, String schemaName) throws Exception {
         }
         return -1l;
     }
+
+    public String executeGetString(String sql, int index) throws SQLException {
+        try( Statement s = getOrCreateConnection().createStatement();
+             ResultSet rs = s.executeQuery(sql))
+        {
+            Assert.assertTrue(rs.next());
+            return rs.getString(index);
```

Comment:
please make this [`null`](https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html#wasNull()) friendly:

```
String result = rs.getString(index);
rs.wasNull() ? null : result;
```

Reply:
this is already covered by getString, see docu https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html#getString(int)

however you're right if we would add sth like `Integer executeGetInteger` we would need to check wasNull since getInt only returns the primitive type `int`.

Reply:
Thanks! I didn't know that it is specified in JDBC interface that reference types must return null if they are null.